src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,35 @@
+.calendar {
+  gap: $gap;
+  width: ($cell-size * 7) + ($gap * 6) + 10px;
+  margin: 0 auto;
+  flex-wrap: wrap;
+  display: flex;
+  align-items: center;
+
+  &__day {
+    background-color: #eee;
+    width: 100px;
+    height: 100px;
+    border: 1px solid black;
+    transition-duration: 500ms;
+
+    &:hover {
+      background-color: pink;
+      cursor: pointer;
+      transform: translateY(-20px);
+      transition-duration: 500ms;
```

Comment:
redundant since you set it for `&__day` (should be only there)
Also, it's a good practice to set proper `transition-property`

Just FYI